Using owncloudcmd from AppImage (#446)
owncloud/docs-client-desktop: modules/ROOT/pages/advanced_usage/command_line_client.adoc

`owncloudcmd` performs a single _sync run_ and then exits the synchronization process. In this manner, `owncloudcmd` processes the differences between client and server directories and propagates the files to bring both repositories to the same state. Contrary to the GUI-based client, `owncloudcmd` does not repeat synchronizations on its own. It also does not monitor for file system changes.

To invoke `owncloudcmd`, you must provide the local and the remote repository URL using the following command, the example is based on Windows:

[source,console]
----
owncloudcmd [options] [arguments]
----

> For the arguments, all of them must be used in the following order, replace the placeholders with real data: 

Reviewer: ```suggestion
Regarding the arguments, all of them must be used in the following order and the placeholders replaced with real data: 
```

== Credential Handling

`owncloudcmd` requires the user to specify the username and password using the standard URL pattern. The example is based on Linux and uses ownCloud Server as backend:

[source,bash]
----

> ./owncloudcmd /home/user/my_sync_folder \
>               https://carla:secret@server/owncloud/remote.php/dav/files/carla/ \
>               /

Reviewer 2: Installed client command can be run from anywhere. 
```suggestion
$ owncloudcmd /home/user/my_sync_folder \
              https://carla:secret@server/owncloud/remote.php/dav/files/carla/ \
              /
```

----

To synchronize the ownCloud directory `Music` to the local directory `media/music`, through a proxy listening on port `8080`, and on a gateway machine using IP address `192.168.178.1`, the command line would be:

[source,console]
----

> ./owncloudcmd --httpproxy http://192.168.178.1:8080 \

Reviewer 2: ```suggestion
$ owncloudcmd --httpproxy http://192.168.178.1:8080 \
```

              $HOME/media/music \

>               https://server/owncloud/remote.php/dav/files/carla \

Reviewer 2: a forward slash (`/`) is MUST at the end of the server url. Otherwise, the command will fail
```suggestion
              https://server/owncloud/remote.php/dav/files/carla/ \
```

              /Music.
----

`owncloudcmd` will prompt for the username and password, unless they have been specified on the command line or `-n` has been passed.

== Exclude List

`owncloudcmd` requires access to an exclude list file. It must either be installed along with `owncloudcmd` and thus be available in a system location, be placed next to the binary as `sync-exclude.lst` or be explicitly specified with the `--exclude` switch.

== Command Line Client When Using AppImages

When using an xref:installing.adoc#appimage[AppImage] installation with Linux, there is no `owncloudcmd` command you can start but the name of the AppImage. To use the command line client of the AppImage, just use the name of the AppImage and append the `--cmd` option plus all other parameters as listed in the xref:#command-description[Command Description]. Replace `[...]` with the version you are using or take the name of the AppImage if you have named it otherwise. Example:

[source,bash]
----
./ownCloud-[...]-x86_64.AppImage --cmd --help

> ----

Reviewer 2: :+1:
